src/cartesi-machine.lua

```diff
@@ -448,7 +446,13 @@ local init_splash = true
 local append_bootargs = ""
 local append_init = ""
 local append_entrypoint = ""
-local rollup
+local rollup = {
```

Comment:
This only solves for `cartesi-machine.lua`, what about the C API `cm_get_default_config` ? Shoudn't we also make this the default there? cc @diegonehab

Reply:
Yes. It should be default in C++ as well.

Reply:
Victor will create an issue about 'get_default_config' vs. 'cartesi-machine.lua'

Reply:
https://github.com/cartesi/machine-emulator/issues/153